examples/can_simple.py

```diff
@@ -34,15 +42,25 @@
         if state == 8: # 8: AxisState.CLOSED_LOOP_CONTROL
             break
 
-# Set velocity to 1.0 turns/s
-bus.send(can.Message(
-    arbitration_id=(node_id << 5 | 0x0d), # 0x0d: Set_Input_Vel
-    data=struct.pack('<ff', 1.0, 0.0), # 1.0: velocity, 0.0: torque feedforward
-    is_extended_id=False
-))
-
-# Print encoder feedback
-for msg in bus:
+# Define an asynchronous listener to handle and print encoder feedback
+def encoder_listener(msg):
     if msg.arbitration_id == (node_id << 5 | 0x09): # 0x09: Get_Encoder_Estimates
         pos, vel = struct.unpack('<ff', bytes(msg.data))
         print(f"pos: {pos:.3f} [turns], vel: {vel:.3f} [turns/s]")
+
+# Control ODrive while notifier object exist
+with can.Notifier(bus, [encoder_listener]):
+    f = 0.5 # Hz
+    torque_ff = 0.0 # torque feedforward
+
+    while True:
+        t = time.time()
+        velocity_setpoint = math.sin(2 * math.pi * f * t) # turns / s
```

Comment:
I do not really comment on the sinusoidal signal we supply the ODrive with, but I think its calculation should be straightforward. I also imported the whole module `math` instead of importing `sin` and `pi`. Looks OK to me.

Reply:
since the math is not what we're showing off, we can compress this into one line. Also use `time.monotonic()` instead of `.time()`.
```suggestion
velocity_setpoint = math.sin(2 * math.pi * 0.5 * time.monotonic()) # turns / s
```